dev/README.md

```diff
@@ -187,11 +187,10 @@ Set proper permissions for the pypirc file:
 chmod 600 ~/.pypirc
 ```
 
-- Install [twine](https://pypi.org/project/twine/) if you do not have it already (it can be done
-  in a separate virtual environment).
+- Install [twine](https://pypi.org/project/twine/) if you do not have it already
 
 ```shell script
-pip install twine
+uv pip install twine
```

Comment:
techincally we could just run `uvx twine`

Reply:
```
❯ uvx twine -h
Installed 22 packages in 165ms
usage: twine [-h] [--version] [--no-color] {check,upload,register}

positional arguments:
  {check,upload,register}

options:
  -h, --help            show this help message and exit
  --version             show program's version number and exit
  --no-color            disable colored output
```

Reply:
oh you already use `uvx` in the next file

Reply:
so we don't need to install twine separately

Reply:
Uvx is for installing tools. Then you run them with just ’twine'. You do not need to use uvx once you install twine

Reply:
yeah `uvx` does require you to install separately, it will take care of it.

Reply:
updated